pkg/gds/mock.go

```diff
@@ -101,6 +114,7 @@ func MockConfig() config.Config {
 			Username: "foo",
 			Password: "supersecretsquirrel",
 			Profile:  "CipherTrace EE",
+			Testing:  true,
```

Comment:
Not sure if the mocked server should be running by default, or if the tests that require it should specify it via a new config.

Reply:
This is a good question - it's kind of a chicken and an egg problem. I suppose I'd rather have Sectigo itself start the mock server if testing is true so that the tests and everything outside don't have to worry about it. We could add a module level `sectigo.Cleanup()` to close and reset the test server. More on that later.